.github/CODEOWNERS

```diff
@@ -0,0 +1,7 @@
+# https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# @global-owner1 and @global-owner2 will be requested for
```

Comment:
```suggestion
# they will be requested for
```
Having the example names here is slightly confusing since they've been replaced with our real usernames below. Other than that, this is great. As @javerbukh said, I didn't  know about this feature - thanks!

Reply:
Good point. Done!